### PR TITLE
Rebuild for Python 3.7, GCC 7, R 3.5.1, openBLAS 0.3.2, Update to 5.5.0

### PR DESCRIPTION
**Pull request by xhochy**
Fixes #9 
Fixes #8 
Fixes #6

### CODE REVIEW

**Comment by xhochy** [comment]
@rmax would be nice if you could have a look at the failing Windows build. From the issue tracker it seems that you fixed the same problem a year ago. You might more easily find the source of the problem.

**Comment by hmaarrfk** [comment]
I think you simply have to deselect the cli tests on windows:
```
py.test --deselect tests/test_cli.py # [win]
py.test                                           # [not win]
```

**Comment by xhochy** [comment]
@conda-forge/ftfy This is ready for review/merge.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -33,7 +33,8 @@ test:
   source_files:
     - tests
   commands:
-    - py.test
+    - py.test --deselect tests/test_cli.py  # [win]
```
Comment (hmaarrfk) [inline recipe/meta.yaml:36]: maybe keep the comment that the cli test fail on windows.

Comment (hmaarrfk) [reply recipe/meta.yaml:36]: thanks.

**Comment by xhochy** [comment]
@hmarrfk This is ready to go.

**Comment by hmaarrfk** [comment]
I can't do anything for you.... unfortunately

@conda-forge/ftfy does anybody want to give this a final review and merge @xhochy's hard work!!!

**Pull request merged**